snmp/metadata.csv

```diff
@@ -89,6 +89,8 @@ snmp.fgFwPol6ByteCount.rate,gauge,,packet,second,[Fortinet FortiGate] The number
 snmp.ifAdminStatus,gauge,,,,[Generic router] [F5 BIG-IP] [Cisco c3850] [Cisco Nexus] [Cisco ASA 5525] The desired state of the interface.,0,snmp,
 snmp.ifSpeed,gauge,,,,"[Generic router] An estimate of the interface's current bandwidth in bits per second, or the nominal bandwidth.",0,snmp,
 snmp.ifHighSpeed,gauge,,,,"[Generic router] An estimate of the interface's current bandwidth in units of 1,000,000 bits per second, or the nominal bandwidth.",0,snmp,
+snmp.ifBandwidthInUsage.rate,gauge,,percent,,"[Generic router] The rate of used received bandwidth in b/s.",0,snmp,
+snmp.ifBandwidthOutUsage.rate,gauge,,percent,,"[Generic router] The rate of used sent bandwidth in b/s.",0,snmp,
```

Comment:
The desc says `in b/s`, but the unit is `percent`.

Should the desc be something like `The percent of input bandwidth used` ? But maybe I'm missing something here.

Reply:
The unit is percent of bit / second, and writing about makes me realise it doesn't make any difference the unit used for rate, as long as total available bitrate and in/out value use the same unit before the rate. Changing it.